addons/airway/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -24,14 +24,50 @@ class ACE_Medical_Treatment_Actions {
         litter[] = {};
         icon = QPATHTOF(ui\larynx.paa);
     };
+    class CancelLarynxtubus {
```

Comment:
why do you create whole new action, when you can just inherit from Larynxtubus

Reply:
because it did not work just coping, like it does not accepts the renaming changes, I can try if it works now tho.